installation/scripts/prom-mtls-patch.sh

```diff
@@ -403,3 +411,42 @@ EOF
   rm testdef.yaml
   rm patchSidecarContainerCommand.yaml
 }
+
+function patchCertManager() {
+  namespace="cert-manager"
```

Comment:
Yes, on a local deployment we're missing the cert-manager deployed. That's why I chose to guard all `kubectl` commands with a `|| true` clause just to make sure that script patches a cert-manager if it exists and it does nothing otherwise. Something smarter would've been go make a guard check in the beginning whether the namespace exists or not and then execute the commands, but hey ... I just figured out that. 😆

Reply:
The istio sidecar injection is actually enabled by default for the `cert-manager` namespace. Part of the code here actually disables injection because some deployments fail when they have a sidecar injected.